panel/src/components/View/Buttons/RemoveLanguageButton.vue

```diff
@@ -0,0 +1,10 @@
+<template>
+	<k-button
+		v-if="$panel.view.props.deletable"
```

Comment:
What happens if the button is used in a view that uses the deletable prop for something completely different?

Reply:
We probably should prepend another if condition that only shows this button on the languages view for now.